geniza/people/translation.py

```diff
@@ -0,0 +1,14 @@
+from modeltranslation.translator import register, TranslationOptions
+from .models import Person, Profession
+
+
+@register(Person)
+class PersonTranslationOptions(TranslationOptions):
+    fields = ("name",)
+    required_languages = ("en",)
+
+
+@register(Profession)
+class ProfessionTranslationOptions(TranslationOptions):
+    fields = ("title",)
+    required_languages = ("en",)
```

Comment:
Ah, interesting. So for every model that is translatable, we have to specify which fields and languages are required? That makes sense. I'm curious how we could simplify it; e.g. maybe a config setting, if the required languages are always the same. But that seems like something we could figure out down the road if and when it's a problem, once we see what the patterns are for how we're using it.